tensorflow_addons/seq2seq/basic_decoder.py

```diff
@@ -111,18 +111,19 @@ def output_dtype(self):
             tf.nest.map_structure(lambda _: dtype, self._rnn_output_size()),
             self.sampler.sample_ids_dtype)
 
-    def step(self, time, inputs, state):
+    def step(self, time, inputs, state, training=None):
```

Comment:
Would `training=False` be better?

Reply:
That's what I initially figured, but keras seems to set `training=None` like [here](https://github.com/tensorflow/tensorflow/blob/545135634295853ad44ef9082a1782f2111db4dd/tensorflow/python/keras/layers/recurrent.py#L676).

Reply:
Yeah, you are right. Find another explanation [here](https://github.com/tensorflow/tensorflow/blob/545135634295853ad44ef9082a1782f2111db4dd/tensorflow/python/keras/engine/training.py#L2658-L2660):
> training: Boolean or None. Only relevant in symbolic mode. Specifies
        whether to build the model's graph in inference mode (False), training
        mode (True), or using the Keras learning phase (None).